lib/preview/routes/proxyPathPageResourceRedirect.js

```diff
@@ -0,0 +1,41 @@
+const fetch = require('node-fetch-commonjs');
+const { isInternalCMSRoute } = require('../previewUtils');
+
+const proxyPathPageResourceRedirect = (req, res, next) => {
+  const isAHtmlRequest = req.accepts().includes('text/html');
+
+  // proxy when referer's path is /proxy and has page query param
+  let proxyPageUrl;
+  let refererUrl;
+  try {
+    refererUrl = new URL(req.headers.referer);
+    proxyPageUrl = new URL(
+      new URL(req.headers.referer).searchParams.get('page')
```

Comment:
Could `new URL(req.headers.referer)` be replaced by `refererUrl `?

Reply:
Oops yes!